backends/LxcBackend.py

```diff
@@ -171,12 +171,10 @@ def configNet(self):
             if 'ipv4' in v:
                 ipv4 = v['ipv4']
                 if not (ipv4 == 'dhcp'):
-                    try:
```

Comment:
In fact we need to keep it, that was a workaround for a bug in the LXC python binding. In some versions we need the block inside the try, in some other we need the except (and this try/catch is ok for that

Reply:
(to be honest, I should add a comment and explicit which is the "modern" case and which is the legacy one, but I currently don't remember... :) )

Reply:
I will add this unprecise comment as an ugly hack when the PR is merged !

Reply:
Ok.
I revert the changes I have made on these try blocks in https://github.com/flesueur/mi-lxc/pull/47/commits/c2f07ab517c91ce32fba07adb43aa86857c8ac4d.